### PR TITLE
163518666 Liking a comment

### PR DESCRIPTION
**Pull request by niomwungeri-fabrice**
#### What does this PR do?
- Like a  comment
#### Description of Task to be completed?
- Enable liking of a comment of an article
#### How should this be manually tested?
- clone the repos and check out the branch `ft-liking-a-comment-163518666`
- clone the repos and check out the branch `develop`
- Run `yarn` to install all dependencies on both branches
- Run `yarn test` to run tests, they should all pass on both branch
- Start your backend server on the address `localhost:3000/` by issuing `yarn start` command
- Create some articles using client tool like the postman 
- Grab slug of one of the articles you created. 
- Run `yarn start` to start your server
- Navigate to `localhost:8080/articles/:{slug}`
- Like on a comment you should see the color of the heart icon change under the article.
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
# 163518649

### CODE REVIEW

**Review by espoirMur — COMMENTED** [review]
Looks great!

**Review by abayo-luc — APPROVED** [review]

**Review by espoirMur — APPROVED** [review]

File: src/__tests__/__actions__/commentsActions.test.js [hunk 1]
```diff
@@ -241,5 +244,51 @@ describe("comments actions", () => {
           });
       });
     });
```
Comment (oesukam) [inline src/__tests__/__actions__/commentsActions.test.js:246]: New line

File: src/__tests__/__reducers__/commentsReducer.test.js [hunk 2]
```diff
@@ -96,4 +101,95 @@ describe("team reducer", () => {
...
+    expect(
+      commentReducer(initialState, {
+        type: SET_LOADING_COMMENTS,
+        isLoading: true
+      })
+    ).toEqual({
+      ...initialState,
+      isLoading: true
+    });
+  });
```
Comment (oesukam) [inline src/__tests__/__reducers__/commentsReducer.test.js:114]: New line

File: src/__tests__/__reducers__/commentsReducer.test.js [hunk 3]
```diff
@@ -96,4 +101,95 @@ describe("team reducer", () => {
       error: "error message"
     });
   });
```
Comment (oesukam) [inline src/__tests__/__reducers__/commentsReducer.test.js:103]: New line

File: src/__tests__/__reducers__/commentsReducer.test.js [hunk 4]
```diff
@@ -96,4 +101,95 @@ describe("team reducer", () => {
...
+      body: "",
+      bodyEdit: "",
+      success: "",
+      error: "",
+      comments: {
+        "9324fbf4-095a-4637-8aff-05d871fc5fc0": { like: 0, liked: false }
+      },
+      isLoading: false
+    });
+  });
```
Comment (oesukam) [inline src/__tests__/__reducers__/commentsReducer.test.js:137]: New line

File: src/__tests__/__reducers__/commentsReducer.test.js [hunk 5]
```diff
@@ -96,4 +101,95 @@ describe("team reducer", () => {
...
+      error: "",
+      comments: {
+        "9324fbf4-095a-4637-8aff-05d871fc5fc0": {
+          id: "9324fbf4-095a-4637-8aff-05d871fc5fc0",
+          body: "hello world"
+        }
+      },
+      isLoading: false
+    });
+  });
```
Comment (oesukam) [inline src/__tests__/__reducers__/commentsReducer.test.js:161]: New line

File: src/__tests__/__reducers__/commentsReducer.test.js [hunk 6]
```diff
@@ -96,4 +101,95 @@ describe("team reducer", () => {
...
+      success: "",
+      error: "",
+      comments: {
+        "9324fbf4-095a-4637-8aff-05d871fc5fc0": {
+          body: "hello world"
+        }
+      },
+      isLoading: false
+    });
+  });
```
Comment (oesukam) [inline src/__tests__/__reducers__/commentsReducer.test.js:185]: New line

File: src/__tests__/__views__/Comments.test.js [hunk 7]
```diff
@@ -194,5 +194,16 @@ describe("Comments", () => {
       mapDispatchToProps(dispatch).onSetBodyEdit(body);
       expect(dispatch.mock.calls[0][0]).toBeDefined();
     });
+
+    test("onLikeComment()", () => {
+      const dispatch = jest.fn();
+      mapDispatchToProps(dispatch).onLikeComment(commentId, slug);
+      expect(dispatch.mock.calls[0][0]).toBeDefined();
+    });
```
Comment (oesukam) [inline src/__tests__/__views__/Comments.test.js:202]: New line

**Review by oesukam — CHANGES_REQUESTED** [review]

**Review by oesukam — APPROVED** [review]

**Pull request merged**